net.sourceforge.jpdftweak.jPdfTweak.json

```diff
@@ -0,0 +1,57 @@
+{
+	"app-id" : "net.sourceforge.jpdftweak.jPdfTweak",
+	"runtime" : "org.freedesktop.Platform",
+	"runtime-version" : "19.08",
+	"sdk" : "org.freedesktop.Sdk",
+	"sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk11" ],
+	"command": "net.sourceforge.jpdftweak.jPdfTweak",
+	"finish-args" : [
+		"--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
+		"--share=ipc",
+		"--socket=x11",
+		"--filesystem=home"
+	],
+	"modules" : [
+		{
+			"name" : "openjdk",
+			"buildsystem" : "simple",
+			"build-commands" : [ "/usr/lib/sdk/openjdk11/install.sh" ]
+		},
+		{
+			"name": "jpdftweak",
+			"buildsystem": "simple",
+			"build-commands": [
+				"unzip jpdftweak-dist.zip -d dist",
+				"mkdir -p /app/jpdftweak",
+				"cp -r dist/* /app/jpdftweak",
+				"for size in 16 32 48 64 128 256; do install -Dm644 /app/jpdftweak/icons/icon${size}.png /app/share/icons/hicolor/${size}x${size}/apps/net.sourceforge.jpdftweak.jPdfTweak.png; done"
+			],
+			"sources": [{
+				"type": "file",
+				"url": "http://prdownloads.sourceforge.net/jpdftweak/jpdftweak-1.1.zip?download",
```

Comment:
This is odd. There's no other way to have a download url which does not contain a query string?

Reply:
Oooops. Query string removed!